draft-ietf-dnsop-svcb-https.md

```diff
@@ -211,20 +211,16 @@ keys. See {{echconfig}}.
 
 ## Terminology
 
-For consistency with {{AltSvc}}, we adopt the following definitions:
+Our terminology is based on the common case where the SVCB record is used to
+access a resource identified by a URI whose `authority` field contains a DNS
+hostname as the `host`.
 
-* An "origin" is an information source as in {{!RFC6454}}.
-  For services other than HTTPS, the exact definition will
-  need to be provided by the document mapping that service
-  onto the SVCB RR.
-* The "origin server" is the server that the client would reach when
-  accessing the origin in the absence of the SVCB record
-  or an HTTPS Alt-Svc.
-* An "alternative service" is a different server that can serve the
-  origin over a specified protocol.
-
-For example within HTTPS, the origin consists of a scheme (typically
```

Comment:
Do we want to keep this illustrative example but reword it?  I think having an example here would help.